### PR TITLE
feat - fix navbar + add languages

### PR DESCRIPTION
**Pull request by rafalbromirski**
### What

Slightly improve `navbar` + add language dropdown.

Not perfect but I had too many constrains from the original theme. I tried not to edit original files (we might have conflicts in the future) - this is the reason I created extra files.

I deployed theme to prod: https://insights.glassnode.com/ (almost no visible diff)

Other improvements:
- fix dark mode toggle (localstorage stores values as string)
- fix `::selection` in light/dark mode
- other fixes / overrides to make it work a bit better

### CODE REVIEW

File: partials/navbar.hbs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="gh-navbar__menu">
+        {{navigation}}
+    </div>
+
+    <div class="gh-navbar__menu">
+        <ul class="nav nav--lang">
+            <li class="nav-lang">
+                <a href="javascript:void()" class="nav-lang-link">Language</a>
+                <ul class="nav-dropdown" aria-label="submenu">
+                    <li><a href="/tag/chinese">中文</a></li>
```
Comment (agudulin) [inline partials/navbar.hbs:22]: wdyt of adding English here as well?

```
<li><a href="/">English</a></li>
```

File: assets/js/dark-mode.js [hunk 2]
```diff
@@ -9,7 +8,7 @@
     var html = document.querySelector("html");
     var toggler = document.querySelector(".js-toggle-mode");
 
-    if (localStorage.getItem(key) || prefersDarkMode.matches) {
+    if (localStorage.getItem(key) === "true" || prefersDarkMode.matches) {
```
Comment (agudulin) [inline assets/js/dark-mode.js:11]: 🙈 🙉

**Review by agudulin — APPROVED** [review]
tested in production, works pretty well 👍

**Comment by Whyseverythingtaken** [comment]
Looks good 👍

**Review by Whyseverythingtaken — APPROVED** [review]

**Pull request merged**